env.example

```diff
@@ -3,6 +3,7 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=3000
+HTTPS=4443
```

Comment:
Let's prefix these both, and update the comment above to mention the difference.

```
HTTP_PORT=3000
HTTPS_PORT=4443
```